pset-functions.js

```diff
@@ -175,6 +257,23 @@ console.log('Problem 4:')
 
 // Add your code below this line
 
+function isPalindrome(word) {
+ const re = /[^A-Za-z0-9]/g
+ word = word.toLowerCase().replace(re, '')
+ const len = word.length
+ for (let i = 0; i < len / 2; i++) {
+   if (word[i] !== word[len - 1 - i]) {
+       return false
+   }
+ }
+ return true
+}
+
+console.log(isPalindrome("noon"))
+console.log(isPalindrome("racecar"))
+console.log(isPalindrome("moon"))
+console.log(isPalindrome("run"))
```

Comment:
@rlou88 Good work here with the isPalindrome function here. You had a great strategy here with your `for loop`, I see you compared the letters by comparing each letter as you loop against the letter on the other side of the string.

example- `Strong`
word[i] !== word[len - 1 - i]
's' == 'g'

Take a look at this solution [here](https://github.com/jsr20190826/hw-02-functions-solution/blob/a4d789161886ff115657133c279062df8a4ca72d/pset-functions.js#L242)